html_css/html_forms.md

```diff
@@ -1,6 +1,6 @@
 ### Introduction
 
-Forms are one of the most important parts of your site. They are your user's gateway into your back end -- the user provides data in a form and you do stuff with it.  Hopefully you've been through the [Rails Course](/ruby-on-rails) and this is all review, but if not make sure to pay attention!
+Forms are one of the most important parts of your site. They are your user's gateway into your back end -- the user provides data in a form and you do stuff with it.  Hopefully you've been through the [Rails Course](https://www.theodinproject.com/courses/ruby-on-rails) and this is all review, but if not make sure to pay attention!
```

Comment:
I think `/courses/ruby-on-rails` is what we need here

Reply:
I had this idea too, but wasn't sure it would work, so pasted an absolute link.

Reply:
Can you change it? A relative path would be better.

Reply:
Yeah, I can. Should I close this pull request and create a new one to do this ?

Reply:
You can if you want, but if you just add another commit to `Northrain-cmd:master` branch it will update this pull request. I can do it for you but I thought I would leave it to you...

Reply:
Thanks, I'll do it myself a copuple of hours later, away from my laptop now. That's an exciting experience for me